PowerEditor/src/winmain.cpp

```diff
@@ -464,11 +465,20 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR pCmdLine, int)
 		nppParameters.setCmdSettingsDir(path);
 	}
 
+	generic_string titleBarAdditional;
+	if (getParamValFromString(FLAG_TITLEBAR_ADD, params, titleBarAdditional))
+	{
+		// could contain double quotes if contains white space
+		if (titleBarAdditional.c_str()[0] == '"' && titleBarAdditional.c_str()[titleBarAdditional.length() - 1] == '"')

```

Comment:
1. Check that `titleBarAdditional` is not empty.
2. You can actually do `titleBarAdditional[0]` or even `titleBarAdditional.front()` and `titleBarAdditional.back()`.
3. What if `titleBarAdditional == "\""`?

Reply:
This is copy/paste/change/don't-think from recent DonHo change involving "settings dir" that suffers from same problems.  :-(

Reply:
Changed.  Thanks @mere-human.